src/bin/tikv-server.rs

```diff
@@ -750,6 +751,22 @@ fn main() {
     // Print version information.
     util::print_tikv_info();
 
+    // Run prometheus client.
+    let prometheus_addr = get_string_value("",
+                                           "metric.prometheus",
+                                           &matches,
+                                           &config,
+                                           Some(String::new()),
+                                           |v| v.as_str().map(|s| s.to_owned()));
+    let prometheus_handler = match prometheus_addr.len() {
```

Comment:
I think we don't need to join this thread, we can let it run forever until program ends.